core/controllers/contributor_dashboard.py

```diff
@@ -126,7 +127,19 @@ class ContributionOpportunitiesHandler(
 
     @acl_decorators.open_access
     def get(self, opportunity_type: str) -> None:
-        """Handles GET requests."""
+        """Handles GET requests and retrieves data based on the
+        opportunity type.
+
+        Args:
+            opportunity_type: str. The opportunity type.
+
+        Raises:
+            PageNotFoundException. The opportunity type is invalid.
+            InvalidInputException. The language_code is invalid.
+                This happens when the opportunity type is of type
+                constant.OPPORTUNITY_TYPE_TRANSLATION and the
+                language_code is set to None by normalized_request.
```

Comment:
Aren't we missing the "Returns" clauses for all these functions?

Reply:
@sagangwee According to the google style guide a return type of None isn't required. Is it required for Oppia?

Reply:
@Helper2020, I don't think we need a return clause for None types, @sagangwee can you please recheck ?

Reply:
Ah, yup. You're right, but then we should update the docstring summary to indicate more clearly what the function does.